### PR TITLE
feat(simtools): `bring simtools`

### PR DESCRIPTION
**Pull request by ekeren**

### CODE REVIEW

File: .github/workflows/canary.yaml [hunk 1]
```diff
@@ -67,6 +67,27 @@ jobs:
       - name: Test
         run: wing test
         working-directory: checks
+  canary-eventbridge:
+    name: Test eventbridge
+    runs-on: ubuntu-latest
+    steps:
```
Comment (ekeren) [inline .github/workflows/canary.yaml:73]: hmm 🤔 , why was this added?

Comment (ekeren) [reply .github/workflows/canary.yaml:73]: @MarkMcCulloh / @Chriscbr ? 

Do you guys know how was this added to my PR? it seems to be created by me running `./mklib.sh simtools`

Comment (eladb) [reply .github/workflows/canary.yaml:73]: Seems like a drift between the script that generates these workflows and `main`. 

@Chriscbr we should add some sort of anti-tamper or self mutation to this repo.

File: simtools/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# simtools
+
+This library is a set of tools that are supposed to make the user of the simulator a happier developer. 
+
+For the time being, it contains only one function, `addMacro`.
```
Comment (eladb) [inline simtools/README.md:5]:
```suggestion
For the time being, it contains only one function, `addMacro()`.
```

Style nit

File: simtools/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## `addMacro`
+
+This function adds a Macro into the target resource.
```
Comment (eladb) [inline simtools/README.md:9]:
```suggestion
This function adds a Macro into the target resource's interaction pane in Wing Console.
```

Comment (eladb) [reply simtools/README.md:9]: Maybe add a small screenshot to demonstrate

File: simtools/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/simtools
+```
+
+## Usage
```
Comment (eladb) [inline simtools/README.md:46]: Seems like a duplicate

**Review by eladb — APPROVED** [review]
Ship it!
